libmetal/lib/processor/hosted/atomic.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * @file	hosted/atomic.h
```

Comment:
hosted or host or vhost?  as it should represent the processor I wonder which is the more appropriate...

Reply:
You just say what you prefer and I write that :)